src/common.rs

```diff
@@ -24,8 +25,11 @@ impl fmt::Display for Warning {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         write!(
             f,
-            "{}:{} {}: {}",
-            self.line.file, self.line.number, self.check_name, self.message
+            "{}:{} {} {}",
```

Comment:
I would change that. 
Since at the moment this code has become unused and `color`/`no-color` is controlled via the argument.

Or I would have thought about how to do formatting at the `Warning` level without special output in the `main.rs`.